### PR TITLE
Add new ${github_notifications} variable to count all unread notifications

### PR DESCRIPTION
**Pull request by lasers**
Testing https://github.com/brndnmtthws/conky/pull/637. :octocat:

### CODE REVIEW

File: src/common.cc [hunk 1]
```diff
@@ -673,6 +674,105 @@ void print_include(struct text_object *obj, char *p, unsigned int p_max_size) {
...
+#define NEW_TOKEN "https://github.com/settings/tokens/new?scopes=notifications&description=conky-query-github\n"
+static size_t read_github_data_cb(char *, size_t, size_t, char *);
+static size_t read_github_data_cb(char *data, size_t size, size_t nmemb, char *p) {
+  char *ptr = data;
+  size_t sz = nmemb * size;
+  size_t z = 0;
+  static size_t x = 0;
+  unsigned int skip = 0;
+
+  for (; *ptr; ptr++, z++) {
```
[the remark this answers is not in the document]
Comment (lasers) [reply src/common.cc]: Fix or proceed? I saw you removing that heart. :sob:

[the remark this answers is not in the document]
Comment (su8) [reply src/common.cc]: :sob: I clicked once nothing happened then I clicked once again and it added heart, dunno what github load/UI was doing.

[the remark this answers is not in the document]
Comment (lasers) [reply src/common.cc]: :sob: Fix or proceed?

[the remark this answers is not in the document]
Comment (su8) [reply src/common.cc]: Proceed ahead :heart_eyes_cat:

**Comment by brndnmtthws** [comment]
There's a bug in Travis that's making it fail to load the sonar credentials correctly because we have 2 identical PRs (one from this repo, one from an external repo). I'm trying to figure out a way to get it to work.

**Pull request closed without merging**